pkg/app/piped/cloudprovider/cloudrun/servicemanifest.go

```diff
@@ -64,6 +64,22 @@ func (m ServiceManifest) YamlBytes() ([]byte, error) {
 	return yaml.Marshal(m.u)
 }
 
+func (m ServiceManifest) AddLabels(labels map[string]string) {
+	if len(labels) == 0 {
+		return
+	}
+
+	lbls := m.u.GetLabels()
+	if lbls != nil {
+		for k, v := range labels {
+			lbls[k] = v
+		}
+	} else {
+		lbls = labels
+	}
+	m.u.SetLabels(lbls)
```

Comment:
nit: We basically follow the early return pattern, but what do you think?
```suggestion
	lbls := m.u.GetLabels()
	if lbls == nil {
		m.u.SetLabels(labels)
		return
	}
	for k, v := range labels {
		lbls[k] = v
	}
	m.u.SetLabels(lbls)
```

Reply:
I like it better because it seems to be Go, but I modified it to match [this](https://github.com/pipe-cd/pipecd/blob/master/pkg/app/piped/cloudprovider/kubernetes/manifest.go#L63).
Which is more appropriate code convention for the PipeCD team?

Reply:
IMO, I don't want to use if-else if I can.

Reply:
I feel like it's better to update the Kubernetes one as well, but it's on you.

Reply:
You're right, we even have this [snipet](https://github.com/pipe-cd/pipecd/blob/master/pkg/app/piped/cloudprovider/kubernetes/manifest.go#L109-L115)
Let's make the return early as convention and change the others place later 👍

Reply:
You can do that by a separate PR.

Reply:
Thanks for all the comments! :) 
I'll fix it. On Other parts, I'll fix in a separate PR.